db/migrate/20180207150921_add_remote_tasks.foreman_tasks.rb

```diff
@@ -0,0 +1,17 @@
+class AddRemoteTasks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :foreman_tasks_remote_tasks do |t|
+      t.string :execution_plan_id, :null => false
+      t.integer :step_id, :null => false
+      t.string :state, :null => false, :default => 'new'
+
+      t.string :proxy_url, :null => false
+      t.string :remote_task_id
+
+      t.index :execution_plan_id
```

Comment:
This index isn't needed, all DBs know how to use partial indices so the second index will also allow using just `:execution_plan_id`.